packages/api/portfolioDrafts/portfolio/createPortfolioStep.test.ts

```diff
@@ -160,10 +136,9 @@ describe("Handler response with mock dynamodb", function () {
 
     const request: APIGatewayProxyEvent = {
       body: JSON.stringify(requestBody),
-      pathParameters: { portfolioDraftId: "1234" },
+      pathParameters: { portfolioDraftId: "aabcbce6-5a91-4a53-bae1-5cf7cae7edd7" },
     } as any;
 
-    const data = await handler(request);
-    expect(data.body).toEqual(`{"code":"OTHER","message":"Database error"}`);
+    await expect(handler(request)).rejects.toThrowError();
```

Comment:
should the `await` be here? or at the invocation to `handler` like:

```
expect(await handler(request)).rejects.toThrowError();
```

Reply:
```
FAIL portfolioDrafts/portfolio/createPortfolioStep.test.ts (14.131 s)
  ● Console

    console.log
      Error [InternalServiceError]: 
          at CommandBehavior.rejects (/home/zclark/atat-developer-clark4/atat-web-api/packages/api/node_modules/aws-sdk-client-mock/src/awsClientStub.ts:184:35)
          at Object.<anonymous> (/home/zclark/atat-developer-clark4/atat-web-api/packages/api/portfolioDrafts/portfolio/createPortfolioStep.test.ts:128:31)
          at Promise.then.completed (/home/zclark/atat-developer-clark4/atat-web-api/packages/api/node_modules/jest-circus/build/utils.js:390:28)
          at new Promise (<anonymous>)
          at callAsyncCircusFn (/home/zclark/atat-developer-clark4/atat-web-api/packages/api/node_modules/jest-circus/build/utils.js:315:10)
          at _callCircusTest (/home/zclark/atat-developer-clark4/atat-web-api/packages/api/node_modules/jest-circus/build/run.js:218:40)
          at processTicksAndRejections (internal/process/task_queues.js:95:5)
          at _runTest (/home/zclark/atat-developer-clark4/atat-web-api/packages/api/node_modules/jest-circus/build/run.js:155:3)
          at _runTestsForDescribeBlock (/home/zclark/atat-developer-clark4/atat-web-api/packages/api/node_modules/jest-circus/build/run.js:66:9)
          at _runTestsForDescribeBlock (/home/zclark/atat-developer-clark4/atat-web-api/packages/api/node_modules/jest-circus/build/run.js:60:9)

      at createPortfolioStepCommand (portfolioDrafts/portfolio/createPortfolioStep.ts:73:13)

  ● Handler response with mock dynamodb › should throw error when unknown dynamodb internal service error occurs

    InternalServiceError:

      126 |
      127 |   it("should throw error when unknown dynamodb internal service error occurs", async () => {
    > 128 |     ddbMock.on(UpdateCommand).rejects({ name: "InternalServiceError" });
          |                               ^
      129 |     // setting up new request
      130 |     const requestBody = {
      131 |       name: "Test portfolio",

      at CommandBehavior.rejects (node_modules/aws-sdk-client-mock/src/awsClientStub.ts:184:35)
      at Object.<anonymous> (portfolioDrafts/portfolio/createPortfolioStep.test.ts:128:31)
```
Using `expect(await handler(request)).rejects.toThrowError();` doesn't work unfortunately. Using the link @juliusfitzhugh-ccpo sent : https://github.com/facebook/jest/issues/6028#issuecomment-567851031 I drew from the answer: 
```
it('terminates the test prematurely', async () => {
  const promise = defuse(new Promise((resolve, reject) => setTimeout(() => reject(new Error('Oops')), 0)));
  await new Promise(resolve => setTimeout(resolve, 10));
  await expect(promise).rejects.toThrow('Oops');
});
```
specifically using `await expect(promise).rejects.toThrow('Oops');` approach.